comp/metadata/systemprobe/def/component.go

```diff
@@ -9,4 +9,6 @@ package def
 // team: agent-shared-components
 
 // Component is the component type.
-type Component interface{}
+type Component interface {
+	GetAsJSON() ([]byte, error)
```

Comment:
Why add this method ?

Reply:
removed in https://github.com/DataDog/datadog-agent/pull/26160/commits/b75328d0788ef1aaf102214124717a445c403d81